### ISSUE TITLE
funbox earthquake/choo choo style mode animation does not work

### ISSUE THREAD

**Issue by someotherotherguy** [Miodec/monkeytype#472]
### Intro
While I was acquainting myself with the various funbox modes, I noticed that the 'earthquake' and 'choo choo' modes do not appear to have any animated or visual effect when selected. I looked at one corresponding style sheet, earthquake.css, and found that the animation is a series of keyframed transform properties, applied to all letter elements:
```css
/* from monkeytype/public/funbox/earthquake.css */

@keyframes shake_dat_ass {
  0% {
    transform: translate(1px, 1px) rotate(0deg);
  }
  ... /* more keyframe data */
  100% {
    transform: translate(1px, -2px) rotate(-1deg);
  }
}

letter {
  animation: shake_dat_ass 0.25s infinite linear;
}
```
### Problem
The issue is that, by default, the letter element is not transformable (i.e. does not use the [box model](https://drafts.csswg.org/css-transforms-1/#transformable-element)), so the transform property does nothing.
### Proposed Solution
The simplest solution is to give the letter element the property `display: inline-block;`. From my testing, this solution does not affect the visuals or layout in any other way.

I would make the change myself, but I am not well acquainted with this project, and I didn't want to go through the whole rigamarole of cloning the repo nor setting up the SCSS preprocessor for one small change. For now, I'll leave it to someone who is more familiar with the appropriate place to make this change.

Hope this helps!

**Comment by Miodec** [comment]
Yeah sorted. This was a problem when I made the letters inline for the arabic ligatures to work. Should be all good now. Thanks

**Comment by someotherotherguy** [comment]
Yeah no problem, I didn't realize how recently this broke. Again I'm new here, so my assumption was that it had been broken for a while. Great to see your dedication to keeping things in tip-top shape, keep it up!

(for others' reference, fixed in commit 2c800a6)
